packages/grafana-ui/src/components/uPlot/config/gradientFills.ts

```diff
@@ -57,35 +58,53 @@ export function getScaleGradientFn(
   }
 
   return (plot: uPlot, seriesIdx: number) => {
+    // A uplot bug (I think) where this is called before there is bbox
```

Comment:
@torkelo will check this out today

Reply:
@leeoniya it only happens for the color used by hover focus point

Reply:
i'll experiment with calling this lazily for hover points, and maybe let's try with using a CSS color gradient for this. (`cursor.points` can already have their own `.fill` and `.stroke` override callbacks which are expected to return css-compatible `background` and `border` values)